app-editors/neovim/neovim-0.7.0.ebuild

```diff
@@ -15,52 +15,56 @@ if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="https://github.com/neovim/neovim.git"
 else
 	SRC_URI="https://github.com/neovim/neovim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="amd64 ~arm ~arm64 x86 ~x64-macos"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~x64-macos"
 fi
 
 LICENSE="Apache-2.0 vim"
 SLOT="0"
-IUSE="+lto +nvimpager +tui"
+IUSE="+lto +nvimpager test +tui"
 
 REQUIRED_USE="${LUA_REQUIRED_USE}"
 # Upstream say the test library needs LuaJIT
 # https://github.com/neovim/neovim/blob/91109ffda23d0ce61cec245b1f4ffb99e7591b62/CMakeLists.txt#L377
-#REQUIRED_USE="test? ( lua_single_target_luajit )"
-#RESTRICT="!test? ( test )"
+REQUIRED_USE="test? ( lua_single_target_luajit )"
+# TODO: Get tests running
+RESTRICT="!test? ( test ) test"
 
 # Upstream build scripts invoke the Lua interpreter
 BDEPEND="${LUA_DEPS}
-	dev-util/gperf
+	>=dev-util/gperf-3.1
 	virtual/libiconv
 	virtual/libintl
 	virtual/pkgconfig
 "
-# TODO: add tests, dev-lua/busted has now got luajit support.
-# bug #584694
+# Check https://github.com/neovim/neovim/blob/master/third-party/CMakeLists.txt for
```

Comment:
Did you remember to do these bits? pkgdiff -b shows some changes. I'll sort it though, minor stuff